pkgs/by-name/po/polarity/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "polarity";
+  version = "unstable-2024-06-28";
+
+  src = fetchFromGitHub {
+    owner = "polarity-lang";
+    repo = "polarity";
+    rev = "59bd7a2c3c3d0a61b25d3bb85b9d21f7b3fef343";
+    hash = "sha256-85uo2GAGxWAWwN2vyhUqwz28Ofb+2eOSuetzovAle+A=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This is IFD, please vendor the lock file.

Reply:
alright :P